commit 14f01bd39866b9d704c7ea69f6b4c45e1350a843
Author: wm4 <wm4@nowhere>
Date:   Wed Oct 18 12:11:37 2017 +0200

aframe: fix logically dead code

Detected by a well known static analyzer.

diff --git a/audio/aframe.c b/audio/aframe.c
index 0a8ec38c93..1f053a6715 100644
--- a/audio/aframe.c
+++ b/audio/aframe.c
@@ -297,7 +297,7 @@ bool mp_aframe_set_chmap(struct mp_aframe *frame, struct mp_chmap *in)

 bool mp_aframe_set_rate(struct mp_aframe *frame, int rate)
 {
-    if (rate < 1 && rate > 10000000)
+    if (rate < 1 || rate > 10000000)
         return false;
     frame->av_frame->sample_rate = rate;
     return true;
